csharp-intermediate-topics/TupleAliasInCSharp/TupleAliasInCSharp/TupleAlias.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace TupleAliasInCSharp;
+
+using EmployeeFinanaceDetails = (int Id, string FName, string LName, double salary);
+
+public class TupleAlias
+{
+    public static void PrintTupleValues()
+    {
+        //ACCESS TUPLE WITH GLOBAL USING DIRECTIVE
```

Comment:
I am not sure the comments are really necessary here, but if we want to keep them (up to your editor @karimkod for sure), we should make them sentence case rather than all uppercase.

Reply:
@AditiSaxena0289 we don't need all these comments. the WriteLine already explains what's happening.